### PR TITLE
2.1.0 - Add nuget package information to release notes

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change adds install and update instructions as well as a recommendation / pointer to NuGet for Unity.

### CODE REVIEW

File: Documentation/ReleaseNotes.md [hunk 1]
```diff
@@ -62,8 +70,42 @@ For the smoothest upgrade path, please use the following steps.
...
+
+1. Select **NuGet > Manage NuGet Packages**
+1. Click the **Update** button for each installed package
+    - Microsoft.MixedReality.Toolkit.Foundation
+    - Microsoft.MixedReality.Toolkit.Tools
+    - Microsoft.MixedReality.Toolkit.Extensions
+    - Microsoft.MixedReality.Toolkit.Examples
+1. Re-open the project in Unity
+
+After updating the packages, messages similar to the following.
```
Comment (keveleigh) [inline Documentation/ReleaseNotes.md]: This feels like an incomplete thought?

```suggestion
After updating the packages, you may see messages similar to the following:
```

File: Documentation/ReleaseNotes.md [hunk 2]
```diff
@@ -62,8 +70,44 @@ For the smoothest upgrade path, please use the following steps.
 
 Related to [issue #6144](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/6144): after upgrading, if you have a custom input simulation profile, the input playback service data provider may have a missing class. Click the "Try Repair" button in the profile window to fix the missing reference.
 
+**Updating fron NuGet**
```
Comment (CDiaz-MS) [inline Documentation/ReleaseNotes.md]:
```suggestion
**Updating from NuGet**
```

**Pull request merged**